corebehrt/classes/normalizer.py

```diff
@@ -0,0 +1,43 @@
+from dask import dataframe as dd
+from corebehrt.functional.normalize import min_max_normalize
+
+
+class ValuesNormalizer:
+    """
+    A class to load normalise values in data frames.
+    Expects 'RESULT' and 'concept' column to be present.
+    """
+
+    @staticmethod
+    def apply_normalization(row, normalization_func, **kwargs):
+        """
+        Applies the given normalisation function if count >= min_count, else sets RESULT to -1.
+        """
+        min_count = kwargs.pop("min_count", 3)
```

Comment:
Shouldn't we just include `min_count=3` as a keyword argument, instead of hiding the dependency in `**kwargs`?